build.sbt

```diff
@@ -10,32 +10,32 @@ name := "couchdb-scala"
 
 version := "0.5.3-SNAPSHOT"
```

Comment:
As you mentioned, since switching to jdk8 is a significant change, I think we should move to 0.6.0-SNAPSHOT at this stage.